dev/breeze/src/airflow_breeze/cache.py

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import sys
+from pathlib import Path
+from typing import Any, List, Tuple
+
+from airflow_breeze import global_constants
+from airflow_breeze.breeze import get_airflow_sources_root
+from airflow_breeze.console import console
+
+BUILD_CACHE_DIR = Path(get_airflow_sources_root(), '.build')
+
+
+def read_from_cache_file(param_name: str) -> str:
```

Comment:
Ah. I found one problem when testing. We need to handle FileNotFoundError: here when ./build/ folder or ./build/.<FILE> is missing.

The .build directory should be created when missing. And missing file should produce a default value.

Reply:
will handle this case @potiuk

Reply:
I have added a check and returned an empty string as default value @potiuk Please share your comments on this and If I could have handled this better, please suggest me.

Reply:
I think it woudl be a bit nicer if `read_from_cache` returned `Optional[str]` and then you could return None in case of missing value.

Reply:
Yes @potiuk I think that would be much apt than returning empty string